src/NHSD.BuyingCatalogue.Solutions.API/Controllers/SolutionsController.cs

```diff
@@ -32,6 +32,21 @@ public SolutionsController(IMediator mediator)
             _mediator = mediator ?? throw new ArgumentNullException(nameof(mediator));
         }
 
+        /// <summary>
+        /// Get a solution matching the specified ID.
+        /// </summary>
+        /// <param name="id">A value to uniquely identify a solution.</param>
+        /// <returns>A task representing an operation to retrieve the details of a Solution.</returns>
+        [HttpGet]
+        [Route("{id}")]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
```

Comment:
`StatusCodes.Status400BadRequest` and `StatusCodes.Status404NotFound`?

I think we should add `[ProducesResponseType(StatusCodes.Status200OK)]` too.

Also, are there definitely any scenarios where this method will return a 400?

Reply:
Good points. Will remove the bad request & add the OK 200.